commands.js

```diff
@@ -1,77 +1,51 @@
 'use strict';
 const fs = require('fs');
-const chalk = require('chalk');
 const path = require('path');
 const os = require('os');
 
 const pathFile = getSavePath();
 
-function getSavePath() {
-  const currentDir = os.homedir();
-  const pth = process.env['SHORTC_PATH'] || currentDir;
-  return path.join(pth, 'shortc.json');
-}
-
-function createIfNotExists(pth) {
+const tryReadFile = async path => {
   try {
-    if (fs.existsSync(pth)) {
-      return;
-    } else {
-      fs.writeFileSync(pth, '[]');
-    }
-  } catch (error) {
-    console.log(error);
+    const file = await fs.promises.readFile(path, { encoding: 'utf8' });
+    return file;
+  } catch (err) {
+    if (err.code !== 'ENOENT') throw err;
```

Comment:
Maybe we should move this cryptic name into more describable variable? Because now I can't understand what kind of error we handle.